sklearn/naive_bayes.py

```diff
@@ -512,8 +512,37 @@ class _BaseDiscreteNB(_BaseNB):
 
     __init__
     _joint_log_likelihood(X) as per _BaseNB
+    _update_feature_log_prob(alpha)
+    _count(X, Y)
     """
 
+    @abstractmethod
+    def _count(self, X, Y):
```

Comment:
I like making this an abstract method. Can we break this PR into two?

- This PR can be for docstring fixes and grammar fixes
- A new PR to only include the abstract methods.

Reducing the scope of PRs makes them smaller so it's easier to review and get merged.

Reply:
Sounds like a good idea, but I don't know how to do that. I made all changes in a single commit, and the best I can do now is to start over. Any advice on how to proceed?

Reply:
The quickest way is:

1. Cut and paste these `abstractmethods` into a empty text file.
2. Commit the changes with the delete abstract methods.
3. Push changes to this PR.
4. Create a new branch lets say "abstractmethods-navie-bayes"
5. Copy and paste the abstract methods to the new branch.
6. Commit, push, and open a new PR with only the abstract methods.

Reply:
Done. Please review.